cpu.js

```diff
@@ -0,0 +1,140 @@
+
+class ComputerPlayer {
+    constructor(){
+        this.index = undefined
+        this.secondIndex = []
+        this.reamingingMove = 6;
+        this.letterGues = []
+        this.letterRight =0;
+    }
+    checkLetter(letter,word){
+        let cLetter = letter // when placed in other class letter will be an arugument 
+        let myL = ''
+        if(!isNaN(Number(cLetter))) { // Checking if word is a number
+            this.index = undefined
+            this.secondIndex = undefined
+            console.log((`This ${cLetter} is not a Valid Letter`))
+        } else {
+        this.letterGues.push(cLetter)
+        console.log("Letters guessed = " + this.letterGues)
```

Comment:
From this project moving forward remove all console logs and debuggers before submitting, encountering them will mean deduction of points